### PR TITLE
#156755804 Update the README

### PR DESCRIPTION
**Pull request by wanjiku-carol**
**What does this PR do?**
Updates the README of the project

**Description of task to be completed?**
- Add  project description
- Add getting started details
- Describe project setup
- Describe database setup

**What are the relevant pivotal tracker stories?**
[#156755804](https://www.pivotaltracker.com/story/show/156755804)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,52 @@
...
+
+> CLIENT_EMAIL
+
+### Dependencies
+- Install the project dependencies:
+> $ pip install -r requirements.txt
+
+### Testing
+- To run tests:
+> $ python manage.py tests
```
Comment (HawiCaesar) [inline README.md]: `python manage.py test` remove the last `s`

File: README.md [hunk 2]
```diff
@@ -1,5 +1,52 @@
...
+# Andela Resource Tracker (ART) Backend
 [![CircleCI](https://circleci.com/gh/AndelaOSP/art-backend/tree/develop.svg?style=svg)](https://circleci.com/gh/AndelaOSP/art-backend/tree/develop)
 [![Coverage Status](https://coveralls.io/repos/github/AndelaOSP/art-backend/badge.svg)](https://coveralls.io/github/AndelaOSP/art-backend)
 
-This is the backend application for ART(Andela Resource Tracker)
+## Description
+This repository contains the API endpoints and models for the ART project implemented using Django. Here in is also the implementation of the admin dashboards using Django templates.
+
+## Setup
+
```
[the remark this answers is not in the document]
Comment (wanjiku-carol) [reply README.md]: This would fall as part of set up right? On it

[the remark this answers is not in the document]
Comment (HawiCaesar) [reply README.md]: I would think so

**Comment by alexkiura** [comment]
@wanjiku-carol I think it would also be helpful to add the step for creating a superuser for logging into the admin.

**Comment by wanjiku-carol** [comment]
@HawiCaesar @alexkiura Done

**Pull request merged**